### ISSUE TITLE
DELETION SUGGESTED: 0845Maqrizi.FadlAlBayt.Shia003726-ara1

### ISSUE THREAD

**Issue by aslishah** [OpenITI/Annotation#709]
This book is a separate edition of a treatise which is part of al-Maqrizi's Rasail compilation. This treatise was also edited under the original title (Kitab Ma‘rifat Ma Yajibu li-Al al-Bayt al-Nabaw min al-Haqq ‘alá Man ‘Adahum) as well as the present title (Fadl Al Bayt); However, I could not find an edition for the current text. While this treatise definitely constitutes a separate risala, its circulation as a separate book is not clear. We also have the Risala in the corpus, which contains this small treatise in its entirely. Perhaps we should delete this one.

**Comment by mabarber92** [comment]
Sorry for the delay here. I just emerged from a rather enormous rabbit hole (I've learned a lot though, so it's all good).

The one collection of Rasail that we have in the corpus is allegedly based on a microfilm of a manuscript in the BNF Paris (4657). The manuscript with that shelfmark in the BNF digital collections does not, however, correspond exactly to the edition (it shares some rasa'il and adds many more, including short texts which probably aren't formally rasa'il). I cannot easily find a manuscript that actually corresponds to the edition we have, but if we are to trust the editor, it has been produced from a microfilmed collection of rasa'il that were written by al-Maqrizi. In short, at some point these rasa'il were collected together into one codex, but it's not clear when. 

The Rasa'il, however, evidently circulated separately and were recompiled in different orders. There is a second manuscript in Leiden (or. 560), which contains Rasa'il that correspond to those found in both the printed edition and the Paris manuscript that I could find. Interestingly, the Risala that I compared had additional material in places (sometimes a few words, sometimes whole sentences), so the Rasa'il must have changed as they circulated as well. According to Bauden, Leiden or. 560 was copied at the request of al-Maqrizi and the manuscript is contemporary to him (https://www.goodreads.com/book/show/22230293-al-maqr-z-s-collection-of-opuscules - unfortunately the blurb is all I can get for this info, as the book has STILL not been published). This means al-Maqrizi himself had envisioned that these Rasa'il should be bundled together into collections.

I would say that we keep the collection of Rasa'il together in the corpus, as it seems to represent a collection that existed at some point (when and at whose behest, given I can't find the actual manuscript, we cannot say). I would argue that we should work towards digitising the other two manuscripts in the future and adding them to the corpus as whole collections. We could use that to tackle this question more fully. (On this it's worth noting that Jo van Steenbergen's edition of the al-Dhahab al-Masbuk , now in the corpus, actually belongs within the Leiden manuscript, and Bauden is working with other scholars to edit and publish other texts from within that manuscript.)

Thank you for leading me on this little journey (it's also helped me find another variation on the fitna narrative as well in one of the Rasa'il collections!). I'll probably write something up on this at some point.
